engine_modules/detroit_inc/models.py

```diff
@@ -116,10 +118,13 @@ def get_dinc_coalition(self, turn=None):
 	if turn is None:
 		turn = self.current_turn
 
-	if turn == 1:
+	# Is that actually OK to do ? What happens when we have an equal vote ? does a session still get put ?
+	# Is that masking the fact that we ask for a turn that doesn't exist (like -1, for example)?
```

Comment:
Asking for a turn that does not exists is a problem. But it should not happen by design, and if it occurs, returning None seems logical